### ISSUE TITLE
Code block syntax highlighting

### ISSUE THREAD

**Issue by brianbunke** [brianbunke/brianbunke.github.io#1]
Post `2017-01-03-get-vmotion.md` highlights the second code block correctly, but not the first. AFAICT, both are invoked the same way.

They both show correctly here, so I guess it's some weirdness with Jekyll/GHP/the theme's CSS.

---

### Excerpt:

Here's how to get up and running:

```posh
# One-time step: Install it from the PowerShell Gallery with PS v5
# Requires an admin PowerShell window
# You may be prompted to add 'C:\Program Files\WindowsPowerShell\Scripts' to environment variable Path
Install-Script 'Get-VMotion'
```

The examples have some more advanced use cases (sorry, I went all-in on the Tron flavoring):

```posh
Get-VMotion
# By default, searches $global:DefaultVIServers (all open Connect-VIServer sessions).
# For all datacenters found by Get-Datacenter, view all s/vMotion events in the last 24 hours.
# VM name, type of vMotion (compute or storage), and vMotion start time & duration are returned by default.
```

**Comment by lidopaglia** [comment]
Hi Brian, 

I took a quick look at this and it seems like the [syntax highlighting style](https://github.com/brianbunke/brianbunke.github.io/blob/master/css/highlightjs.piperita.scss) of your theme might be causing you some trouble.

I inserted your post into a fresh Jekyll instance with the base Minima theme and it [looks closer](https://cloud.githubusercontent.com/assets/1696777/26430796/f8a3b788-40bd-11e7-9374-efd2a20d136e.png) to what you'd expect it to.

Also, rouge's [powershell lexer](https://cloud.githubusercontent.com/assets/1696777/26430796/f8a3b788-40bd-11e7-9374-efd2a20d136e.png) only has a finite list of "builtins" which does not include support for custom cmdlet names or cmdlets like `Get-Cluster` or `Get-VM`.

There's also an [open PR](https://github.com/jneen/rouge/pull/660) for resolving an issue with the incorrect handling of escape characters in the powershell lexer.

Lastly, it may help to include a Gemfile in your Jekyll project and specify using the GitHub Pages gem to build and test [your site locally](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/). By [using the GitHub Pages gem](https://jekyllrb.com/docs/github-pages/#use-the-github-pages-gem) you'll ensure you're using the same versions of jekyll, kramdown, rouge, etc. as GitHub will when building your site.

I hope that helps somewhat.

**Comment by brianbunke** [comment]
Dude. 🤗 This is excellent. Gives me a great starting place to investigate this weekend.

Thank you, Lido!

**Comment by brianbunke** [comment]
Current status & info for my future self. I'm hoping to carve out time in mid-June to resolve this.

- Luckily, my [current setup process](http://www.brianbunke.com/blog/2017/05/24/jekyll-win10/) appears to render the same as GitHub Pages in my tests, and the days of five blind, worthless commits against `master` (see above) are gone
- [rouge](https://github.com/jneen/rouge) & its [PowerShell lexer](https://github.com/jneen/rouge/blob/master/lib/rouge/lexers/powershell.rb)
- I may want to drop rouge completely, and investigate using Minima's highlighting with this theme
  - That example pic is pretty appealing as a low-maintenance option

**Comment by brianbunke** [comment]
Mostly fixed in e88efe1 by moving to [Minima's syntax highlighting](https://github.com/jekyll/minima/blob/affcd93be22e960afd2be08e6891d99b31bcf920/_sass/minima/_syntax-highlighting.scss). Things not fixed:

- Custom commands (`Fake-CommandName`) won't highlight
    - Would be nice to add local support where I didn't have to pollute an upstream lexer, but not sure if that's a thing
- Bottom of [this page](http://www.brianbunke.com/blog/2017/08/17/confluenceps-2/), the code block chokes on `Set-` when I give it a command name it obviously doesn't know

But I'm not worried about those problems, because I'm savoring the victory.

Thanks again for taking the time to point me in the right direction, @lidopaglia 👏

**Comment by lidopaglia** [comment]
No probs. 👍
